templates/flavors/base/linodeVPC.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+kind: LinodeVPC
+metadata:
+  name: vpc-${CLUSTER_NAME}
```

Comment:
can we drop the vpc from the name of this? if we want to call it something other than just `cluster_name` we could use some descriptor to mention it is for cluster networking, but I would also be ok with just `$CLUSTER_NAME`

Reply:
Updated, added `VPC_NAME` flag. If the flag is not set, it uses `CLUSTER_NAME`